ci/github/get_hdf5_if_needed.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+# vendored from https://github.com/h5py/h5py/blob/master/ci/get_hdf5_if_needed.sh
```

Comment:
Please check if this license of this script is already included in LICENSES. You should to add it otherwise.

Reply:
h5py is already part of the licenses here: https://github.com/PyTables/PyTables/blob/master/LICENSES/H5PY.txt

I think this will cover this script, too (it's not explicitly licensed otherwise as far as i can see).

Reply:
OK, thanks